src/dev/lupio/LupioRTC.py

```diff
@@ -0,0 +1,19 @@
+from m5.params import *
+from m5.proxy import *
+from m5.objects.Device import BasicPioDevice
+from m5.util.fdthelper import *
```

Comment:
Double check that all of these imports are necessary.

Reply:
To add, avoid `import *` if you can.